### PR TITLE
Integrate register input service

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
- Recording events from keyboard mouse and virtualkeyboard including touch events
- Introduced tests

### CODE REVIEW

File: pupil-spa/src/app/check/check.component.ts [hunk 1]
```diff
@@ -18,6 +18,14 @@ export class CheckComponent implements OnInit {
   public question: Question;
   public config: Config;
 
+  // @HostListener('document:keydown', ['$event'])
```
Comment (jon-shipley) [inline pupil-spa/src/app/check/check.component.ts]: I think we still need this

Comment (GeorgeChatzigiannis) [inline pupil-spa/src/app/check/check.component.ts]: @jon-shipley introduced back

File: pupil-spa/src/app/check/check.component.spec.ts [hunk 2]
```diff
@@ -74,20 +74,14 @@ describe('CheckComponent', () => {
     spyOn(mockQuestionService, 'getNextQuestionNumber').and.returnValue(null);
     component.nextQuestion();
     fixture.whenStable().then(() => {
-      const compiled = fixture.debugElement.nativeElement;
+      // const compiled = fixture.debugElement.nativeElement;
```
Comment (jon-shipley) [inline pupil-spa/src/app/check/check.component.spec.ts]: Do we need to leave this commented out?

Comment (GeorgeChatzigiannis) [inline pupil-spa/src/app/check/check.component.spec.ts]: @jon-shipley removed test

**Pull request merged**